integration-tests/kamelet/src/main/resources/application.properties

```diff
@@ -15,3 +15,8 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 camel.kamelet.setBodyFromProperties.bodyValueFromProperty=Camel Quarkus Kamelet Property
+
+quarkus.camel.kamelet.identifiers = injector,logger
+
+# this is needed to actually test that kameleta are preloaded at build time:
```

Comment:
kameleta : typo ?

Reply:
yep tnx!

Reply:
looks like a good name, let's remember this for some new features